src/Forms/index.ts

```diff
@@ -1,2 +1,3 @@
 export { default as ValidationErrorMessage } from './ValidationErrorMessage';
 export { default as TextInput } from './TextInput';
+export { default as Label } from './Label';
```

Comment:
One of our design decisions early on was that `Label` should not be exported from the library, since we don't want developers to have the options to use a `<Label>` on its own without associating it with a field.

Reply:
Sorry, forgot about this! I removed export through commit a716905.